### ISSUE TITLE
Jumping is very weak in "Dash" jetpack mode

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#5764]
### Short Description of the issue:
When in this jetpack mode
![image](https://github.com/tsunamayo/Starship-EVO/assets/27857711/4ebbf7f4-abc0-441e-be22-2a6bb46f9dc9)
Jumping is very weak, about 1/5th of the strength of this jetpack mode
![image](https://github.com/tsunamayo/Starship-EVO/assets/27857711/4ccef398-f63a-485f-bb65-bf4710cdb672)

https://github.com/tsunamayo/Starship-EVO/assets/27857711/95ebf45d-463f-48fa-a24e-9bff2406c4bd

As an aside, it might be better to switch to a walking icon when in the second mode to better convey that there is no jetpack assist. The difference between the grey and white colour that the icon changes to currently is not very noticeable, and it only conveys information when you change the mode. If you just glace down at it while in walking mode you still see a jetpack in the icon 
![Health__2861_29](https://github.com/tsunamayo/Starship-EVO/assets/27857711/83bf485c-ad29-4141-b007-a170c15b5494)

### Steps to reproduce the issue:
- Switch to the designated jetpack mode and jump

### Environment
- Starship EVO version: 24w20a
- Graphics Card: RX 6700XT

**Comment by tsunamayo** [comment]
@ProPeach Hi, yeah I agree with the gray out thing, it always bugged me too. For the jetpack well it is weak because it is not a jump, just a weak jetpack push. I think I need some more serious tweak to the jetpack, still not satisfied at all.

**Comment by ProPeach** [comment]
Yeah I wondered if it might be something like that. I suppose in that case, why have the avatar use the jetpack when you hit space if the jump is stronger?

**Comment by tsunamayo** [comment]
@ProPeach Well because when you press jump you trigger the jetpack push, except because you release jump too soon you get a mini jump. If the jetpack was as strong as the jump you would be able to fly to space.
